server/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -733,6 +729,32 @@ private void updateParentsInnerTypes(ColumnIdent column,
         }
     }
 
+    public DocTableInfo dropConstraint(String constraint) {
+        List<CheckConstraint<Symbol>> newConstraints = checkConstraints.stream()
+            .filter(x -> !x.name().equals(constraint))
+            .toList();
```

Comment:
Minor: doesn't filter go through all elements even when the one is found? I think we only need to remove the first element with the name.

Reply:
Yes but the new constraints need to include everything not removed